susemanager/bin/mgr-setup

```diff
@@ -46,6 +46,10 @@ if [ ! $? -eq 0 ]; then
     exit 1
 fi
 
+if [[ ! -z "$TZ" ]]; then
```

Comment:
Just curious, why are you using negative `! -z "$TZ"` instead of `-n "$TZ"`?

Reply:
actually no reason, probably it's clear `-n` . I changed it